src/www/app/main.ts

```diff
@@ -96,12 +97,13 @@ export function main(platform: OutlinePlatform) {
       const debugMode = queryParams.get('debug') === 'true';
 
       const eventQueue = new EventQueue();
+      const uiService = new OutlineUIService();
```

Comment:
Revert.
Handle the system not configured before you call start() (by checking the set up status) or by handling a not set configured error thrown by start().
You then call a method to configure/set up the VPN before calling start again if setup was successful.